tests/api.c

```diff
@@ -60917,6 +60917,7 @@ static void test_wolfSSL_dtls12_fragments_spammer(WOLFSSL* ssl)
 #ifdef WOLFSSL_DTLS13
 static void test_wolfSSL_dtls13_fragments_spammer(WOLFSSL* ssl)
 {
+    const int sendCountMax = 100;
```

Comment:
Please change to `word16` so that no compilers will complain about the different types in the `for()` comparison.

Reply:
Changed it.